screens/Home/index.tsx

```diff
@@ -101,10 +104,14 @@ const styles = StyleSheet.create({
     width: 250,
     marginRight: 10,
   },
-  cardTitle: {
+  lightCardTitle: {
     fontSize: 16,
     color: Colors.light.gray900,
   },
+  darkCardTitle: {
+    fontSize: 16,
+    color: Colors.light.white,
+  },
```

Comment:
Keep this as `cardTitle` but remove the color property. It's a bit awkward how you've used `Colors.light.white` for a `darkCardTitle`. Instead you should use the hook to pluck out the `textColor` which is automatically white when the phone is in a light theme :) 

```suggestion
  cardTitle: {
    fontSize: 16,
  },
```